desktop/app.js

```diff
@@ -83,6 +84,11 @@ module.exports = function main() {
       // when you should delete the corresponding element.
       mainWindow = null;
     });
+
+    // wait until window is presentable
+    mainWindow.once('ready-to-show', function() {
+        mainWindow.show()
```

Comment:
Would you mind throwing a semicolon at the end here? Just want to try to match the coding style with the rest of the file.

Reply:
On that note a run through of `prettier` should take care of any and all styling issues.